vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -83,6 +83,10 @@
      */
     public DatePicker() {
         this((LocalDate) null, true);
+        // Trigger model-to-presentation conversion in constructor, so that
+        // the client side component has a correct initial value of an empty
+        // string
```

Comment:
Can't remember anymore if/why this was an appropriate wording for Select, but maybe we should rather hint at the fact that this avoids the initial value-changed event from the WC?

Reply:
Decided to fix the issue in `Vaadin Generated DatePicker`. The comment is no longer necessary as the fix looks natural there. Let me know if you think it is still needed.